### PR TITLE
Add unit test for shardingsphere-infra-binder

### LINKED ISSUE #10971: Add unit test for shardingsphere-infra-binder
Hi community,

This issue is for #10887.

## Aim
Add unit test for shardingsphere-infra-binder project to test its public functions.

## Basic Qualifications
- Java  
- Maven  
- Junit.Test

## Classes

Add unit test for classes to test its public functions.

- [ ] AnalyzeTableStatementContext
- [ ] ExplainStatementContext
- [ ] ShowColumnsStatementContext
- [ ] ShowCreateTableStatementContext
- [ ] ShowIndexStatementContext
- [ ] DenyUserStatementContext
- [ ] GrantStatementContext
- [ ] RevokeStatementContext
- [ ] AlterIndexStatementContext
- [ ] AlterTableStatementContext
- [ ] AlterViewStatementContext
- [ ] CreateDatabaseStatementContext
- [ ] CreateFunctionStatementContext
- [ ] CreateIndexStatementContext
- [ ] CreateProcedureStatementContext
- [ ] CreateTableStatementContext
- [ ] CreateViewStatementContext
- [ ] DropIndexStatementContext
- [ ] DropTableStatementContext
- [ ] DropViewStatementContext
- [ ] PrepareStatementContext
- [ ] TruncateStatementContext

## Examples FYI
- CallStatementContextTest

### PR DESCRIPTION
**Pull request by sydgeek**
Fixes #10971.

### CODE REVIEW

**Comment by tristaZero** [comment]
Hi @sydgeek Welcome, but could you firstly look at the failed CI test?

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi @sydgeek Welcome, but could you firstly look at the failed CI test?

File: shardingsphere-infra/shardingsphere-infra-binder/src/test/java/org/apache/shardingsphere/infra/binder/statement/dal/ExplainStatementContextTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        assertThat(actual.getDatabaseType().getName(), is("MySQL"));
+    }
+
+    @Test
+    public void assertPostgreSQLNewInstance() {
+        ExplainStatementContext actual = assertNewInstance(mock(PostgreSQLExplainStatement.class));
+        assertThat(actual.getDatabaseType().getName(), is("PostgreSQL"));
+    }
+
+    private ExplainStatementContext assertNewInstance(final ExplainStatement explainStatement) {
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-binder/src/test/java/org/apache/shardingsphere/infra/binder/statement/dal/ExplainStatementContextTest.java:50]: It seems method `assertXXX` should not need return value

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-infra/shardingsphere-infra-binder/src/test/java/org/apache/shardingsphere/infra/binder/statement/dal/ExplainStatementContextTest.java (hunk 1 above)
Comment (sydgeek) [reply shardingsphere-infra/shardingsphere-infra-binder/src/test/java/org/apache/shardingsphere/infra/binder/statement/dal/ExplainStatementContextTest.java:50]: The test classes I submitted all have this problem, so let me change it together.

**Comment by sydgeek** [comment]
@tristaZero Hi,problem solved.

**Comment by tristaZero** [comment]
Hi @sydgeek thanks.

**Review by tristaZero — APPROVED** [review]

**Review by terrymanu — APPROVED** [review]

**Pull request merged**